notebooks/WET/WET-004/WET-004-ComCam.ipynb

```diff
@@ -0,0 +1,385 @@
+{
```

Comment:
<p>A bit underwhelmed with the comparison. I'd honestly expect that the closer the blend, the worse the performance.  Didn't we take ComCam data in fields with multiple blends per CCD? Please point to a relevant dayObs / seqNum and I can rerun these cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<img width="566" alt="image" src="https://github.com/user-attachments/assets/ce3f64ae-d1af-4c78-b381-810d6cd43f3e" />